molviewspec/molviewspec/builder.py

```diff
@@ -98,7 +107,51 @@ def additional_properties(self, **kwargs) -> Self:
         return self
 
 
-class Root(_Base):
+class _PrimitivesMixin:
+    def primitives(
+        self,
+        *,
+        default_color: ColorT | None = None,
+        default_label_color: ColorT | None = None,
+        default_tooltip: str | None = None,
+        transparency: float | None = None,
```

Comment:
I moved the common parameters for the primitives group to the node itself. Again, this is to streamline both the builder use and the JSON data use.